src/main/scala/com/spotify/spark/bigquery/BigQueryClient.scala

```diff
@@ -109,8 +110,29 @@ private[bigquery] class BigQueryClient(conf: Configuration) {
    */
   def load(gcsPath: String, destinationTable: TableReference,
            writeDisposition: WriteDisposition.Value = null,
-           createDisposition: CreateDisposition.Value = null): Unit = {
+           createDisposition: CreateDisposition.Value = null,
+           isPartitionedByDay: Boolean = false): Unit = {
```

Comment:
Yes that is correct. As it stands this is the only way

We can add it, but wouldnt it be more explicit this way? Otherwise a developer will have to look up potential options from CreateDisposition and it isnt very explicit or intuitive

That said if I was entirely new to the code base, I would either look for a helper or the signature of the saveToBigQuery method 

Thoughts?

You guys are the maintainers so happy to do it however you wish :)